src/explore-education-statistics-common/src/components/ContentSectionIndex.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import findAllForId from '@common/lib/dom/findAllForId';
+
+interface IndexResult {
+  element: Element;
+  location: string;
+}
+
+interface Props {
+  fromId: string;
+}
+
+interface State {
+  indexResults: IndexResult[];
+  indexComplete: boolean;
+}
+
+class ContentSectionIndex extends Component<Props> {
+  public state: State = {
+    indexResults: [],
+    indexComplete: false,
+  };
+
+  public componentDidMount() {
+    const { fromId } = this.props;
+    this.index(fromId);
+  }
+
+  private static getLocationText(element: HTMLElement): string {
+    const location: string[] = [];
+
+    element.tagName.toLowerCase();
```

Comment:
Don't think this is actually doing anything?

Reply:
agreed - removed.